mlrun/feature_store/api.py

```diff
@@ -98,6 +98,7 @@ def get_offline_features(
     update_stats: bool = False,
     engine: str = None,
     engine_args: dict = None,
+    filter: str = None,
```

Comment:
`filter` is a reserved word in python, I'm not sure its a healthy code to override it.